junit-runtime/src/main/scala/com/novocode/junit/OutputCapture.scala

```diff
@@ -0,0 +1,47 @@
+package com.novocode.junit
+
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+object OutputCapture {
+  def start(): OutputCapture = new OutputCapture()
+
+  private def getScalaOut(): PrintStream = {
+    try {
+      scala.Console.out
+    } catch {
+      case t: Throwable => null
```

Comment:
Oh, there's catching Java reflection error. There cannot be any reflection error in our case since we don't use reflection. So you can remove the try/catch altogether.